### PR TITLE
Update twilio rates

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?183754

opening for tests

### CODE REVIEW

**Comment by nickpell** [comment]
@orangejenny

File: corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py [hunk 1]
```diff
@@ -12,16 +12,15 @@
...
     currency_class = apps.get_model('accounting', 'Currency') if apps else Currency
     sms_gateway_fee_class = apps.get_model('smsbillables', 'SmsGatewayFee') if apps else SmsGatewayFee
     sms_gateway_fee_criteria_class = apps.get_model('smsbillables', 'SmsGatewayFeeCriteria') if apps else SmsGatewayFeeCriteria
 
     # iso -> provider -> rate
     def get_twilio_data():
-        twilio_file = open('corehq/apps/smsbillables/management/'
-                           'commands/pricing_data/twilio-international-sms-rates.csv')
-        twilio_csv = csv.reader(twilio_file)
+        twilio_file = open(twilio_rates_filename)
```
Comment (orangejenny) [inline corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py]: might as well use a `with`

File: corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py [hunk 2]
```diff
@@ -12,16 +12,15 @@
...
+        twilio_csv = csv.reader(twilio_file.read().splitlines())
```
Comment (orangejenny) [inline corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py]: not following why splitlines is necessary now - doesn't `reader` take in a file and return an iterable?

Comment (orangejenny) [inline corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py]: nm, didn't realize the csv is now a single line. odd.

File: corehq/apps/smsbillables/management/commands/pricing_data/twilio-rates-2015_10_06.csv [hunk 3]
```diff
@@ -0,0 +1 @@
...
ZA,South Africa,SOUTH AFRICA Outbound SMS - Other,0.02167
ZA,South Africa,SOUTH AFRICA Outbound SMS - Vodacom,0.02167
ZM,Zambia,ZAMBIA Outbound SMS - Airtel,0.02167
ZM,Zambia,ZAMBIA Outbound SMS - MTN,0.04767
ZM,Zambia,ZAMBIA Outbound SMS - Other,0.02167
ZM,Zambia,ZAMBIA Outbound SMS - ZAMTEL,0.04
ZW,Zimbabwe,ZIMBABWE Outbound SMS - Econet Wireless,0.11483
ZW,Zimbabwe,ZIMBABWE Outbound SMS - NetOne,0.013
ZW,Zimbabwe,ZIMBABWE Outbound SMS - Other,0.013
ZW,Zimbabwe,ZIMBABWE Outbound SMS - Telecel,0.0195
```
Comment (orangejenny) [inline corehq/apps/smsbillables/management/commands/pricing_data/twilio-rates-2015_10_06.csv]: just checking - this deals with unicode, right? was looking at slovenia.

**Comment by orangejenny** [comment]
looks fine, just want verification that unicode in the csv is fine

File: corehq/apps/smsbillables/management/commands/pricing_data/twilio-rates-2015_10_06.csv (hunk 3 above)
Comment (nickpell) [inline corehq/apps/smsbillables/management/commands/pricing_data/twilio-rates-2015_10_06.csv]: I was able to run the migration locally without running into problems from the unicode

**Pull request merged**